### PR TITLE
Nonfunctional changes to improve consistency and clarity

### PR DESCRIPTION
**Pull request by spawnia**

### CODE REVIEW

File: tests/Executor/DeferredFieldsTest.php [hunk 1]
```diff
@@ -401,15 +401,15 @@ public function testComplexRecursiveDeferredFields() : void
                 return [
                     'sync'         => [
                         'type'    => Type::string(),
-                        'resolve' => function ($v, $a, $c, ResolveInfo $info) {
+                        'resolve' => function ($val, $args, $context, ResolveInfo $info) {
```
Comment (simPod) [inline tests/Executor/DeferredFieldsTest.php]: I think we can use `value` and `arguments` if we are doing this already :)

File: src/Executor/ReferenceExecutor.php [hunk 2]
```diff
@@ -927,28 +928,27 @@ function ($previous, $value) use ($callback) {
...
-     * Complete a list value by completing each item in the list with the
-     * inner type
+     * Complete a list value by completing each item in the list with the inner type.
      *
-     * @param FieldNode[] $fieldNodes
-     * @param mixed[]     $path
-     * @param mixed       $result
+     * @param FieldNode[]         $fieldNodes
+     * @param mixed[]             $path
+     * @param mixed[]|Traversable &$results
```
Comment (simPod) [inline src/Executor/ReferenceExecutor.php]: is specifying `&` in phpdocs standard thing? I'm not against just don't remember seeing it anywhere else 🤔

Comment (spawnia) [reply src/Executor/ReferenceExecutor.php]: It is not in the standard. Removed

File: tests/Executor/DeferredFieldsTest.php (hunk 1 above)
Comment (spawnia) [reply tests/Executor/DeferredFieldsTest.php]: `args` is used anywhere, and `$val` is going to be `$rootValue`

**Pull request merged**